### PR TITLE
Support all active Julia 1.x (and test with them)

### PR DESCRIPTION
**Pull request by tkf**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,10 @@ os:
   - linux
   - osx
```
Comment (tkf) [inline .travis.yml]:
```suggestion
```

I highly doubt this package behaves differently in different OSes. So, how about not testing with macOS to speedup CI?

**Comment by jw3126** [comment]
LGTM

File: .travis.yml (hunk 1 above)
Comment (jw3126) [reply .travis.yml]: :+1: I also think 1.0 + stable + nightly julia version would suffice.

**Comment by tkf** [comment]
@jw3126 Can you check the comment I just posted above https://github.com/JuliaObjects/ConstructionBase.jl/pull/8#discussion_r329753900?

**Pull request merged**